examples/byte-type/byte_type.description

```diff
@@ -1,3 +1,3 @@
-//Byte is 8 bit and it is just shorthand for 0|1|2|...|255 in ballerina.
-//Along with general byte array type, there is also a special syntax for defining base64 based array of bytes.
-//This example demonstrates how byte and byte array can be used with ballerina programs.
+// The `byte` type is just shorthand for 0|1|2|...|255.
+// Along with the general `byte` array, there is also a special syntax to define `base16` and `base64` based arrays of
+// bytes. This example demonstrates how `byte` and `byte` array can be used in Ballerina programs.
```

Comment:
Can we also mention why byte and byte arrays are used? I think it might be useful to a reader to understand the use case as well.

Reply:
Will re-visit in the next iteration. IMO, if we are adding this information, we would have to update all the BBEs in a similar manner.